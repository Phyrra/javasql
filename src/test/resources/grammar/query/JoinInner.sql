select 1
from a
inner join b on b.id = a.id