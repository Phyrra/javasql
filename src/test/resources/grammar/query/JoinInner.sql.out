SELECT 1
FROM [a]
INNER JOIN [b] ON [b].[id] = [a].[id]